src/main/java/spoon/support/compiler/jdt/JDTBasedSpoonCompiler.java

```diff
@@ -95,13 +95,8 @@ public JDTBasedSpoonCompiler(Factory factory) {
 	}
 
 	private void initializeCUCOmparator() {
-		try {
-			this.sortList = System.getenv("SPOON_SEED_CU_COMPARATOR") == null;
-		} catch (NumberFormatException | SecurityException e) {
-			Launcher.LOGGER.error("Error while parsing Spoon seed for CU sorting", e);
-			this.sortList = true;
-		}
-
+		this.sortList = System.getenv("SPOON_SEED_CU_COMPARATOR") == null;
+		this.sortList = true;
```

Comment:
Why this.sortList should always be true? I guess you forgot to remove this line.